sage/api/src/error.rs

```diff
@@ -1,7 +1,7 @@
 pub enum TransitionError {
-	InvalidTransitionId,
```

Comment:
I do not think that this error makes sense, rust's type system makes it impossible to mess this up.